docs/plugins/introduction.mdx

```diff
@@ -171,3 +178,59 @@ No, pre-parse plugins cannot modify the request in the execution pipeline. The p
 pipeline by returning appropriate responses.
 
 :::
+
+## Pre-Response Plugin
+
+The `pre-response` plugin is triggered at the final step in the execution pipeline, after the query is executed. Use this
+step to add custom logic after the query is executed but before the response is sent to the client.
```

Comment:
let's clarify the "add custom logic" part here. Right now this could be misunderstood as - the user has the ability to augment /change the response before sending to the client